styleguide.config.js

```diff
@@ -28,5 +28,6 @@ module.exports = {
     'holderjs',
     'assets/stylesheets/base.scss',
     path.join(__dirname, 'styleguide_assets/rcl_styles.module.scss')
-  ]
+  ],
+  pagePerSection: true,
```

Comment:
This is a pretty nice quality-of-life change, it makes the page load _soo_ much faster, because each component's docs are on its own page rather than being concatenated into one humongous page that takes a while to load. Check out [the docs](https://github.com/styleguidist/react-styleguidist/blob/master/docs/Configuration.md#pagepersection)

Reply:
I really like this change! You might want to drop a quick note to the design team about this change when it gets merged in as well, only because they sometimes reference the RCL.

Reply:
great change here, thanks!!!